recipes/main/thai_red_curry.md

```diff
@@ -0,0 +1,41 @@
+# Thai red curry
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-30%25-green.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-1.5hr-orange.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Average-yellow.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+**Head chef:** [@matthewdking](https://github.com/matthewdking/)
+
+**Sous chef:** [@Karyum](https://github.com/Karyum/)
+
+**Date:** 29/01/18, Week 13 FACN3
+
+**Number of servings:** 8
+
+**Author of Recipe**: [@matthewdking](https://github.com/matthewdking/)
+
+## Ingredients
+
+* 2 onions, sliced
+* 4 small sweet potatoes, peeled and chopped into about 2cm cubes
+* 1 large aubergine, chopped into about 2cm cubes
+* 2 red peppers, sliced
```

Comment:
they were not official in the recipe you decided to add them for no reason

Reply:
But I made the recipe up as I went along so its part of the recipe

Reply:
well i give this recipe a bad review from me cause actually i didn't like the peppers in it :smirk:

Reply:
are you suggesting I should change the suggestion badge?

Reply:
@finnhodgkin #56